ui/src/components/homepage/Easter.js

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import {
+    Flex, 
+    Button, 
+    Text,
+    Stack,
+    Link,
+    SimpleGrid
+  } from '@chakra-ui/react';
+import { AiOutlineDesktop, AiOutlineInstagram } from 'react-icons/ai';
+import { GiPartyPopper } from 'react-icons/gi';
+import { useMediaQuery } from '@chakra-ui/media-query';
+
+const HeroSection = () => {
+  const [isNotSmallerScreen] = useMediaQuery('(min-width:648px)');
```

Comment:
you don't have to change it if you don't have time, but a lot of this code is duplicated in GoodFriday.js, would be better to re-use that same exact file and just handle if it's GoodFriday mode/Easter mode